modules/ocf_kubernetes/manifests/master/ingress/nginx.pp

```diff
@@ -2,8 +2,6 @@
 
   $kubernetes_worker_nodes = lookup('kubernetes::worker_nodes')
   $kubernetes_workers_ipv4 = $kubernetes_worker_nodes.map |$worker| { ldap_attr($worker, 'ipHostNumber') }
-  $nginx_version = lookup('kubernetes::nginx_version')
-  $ingress_nginx_url = "https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v${nginx_version}/deploy/static/provider/baremetal/deploy.yaml"
```

Comment:
We need some way of actually tying our manifests to upstream...
Since the hiera key is not used, it should be removed, and we should maintain a patch/diff relative to upstream instead of having an opaque modified blob.